src/main/java/com/microtripit/mandrillapp/lutung/model/MandrillRequestDispatcher.java

```diff
@@ -61,7 +60,14 @@
 				.setSocketTimeout(SOCKET_TIMEOUT_MILLIS)
 				.setConnectTimeout(CONNECTION_TIMEOUT_MILLIS)
 				.setConnectionRequestTimeout(CONNECTION_TIMEOUT_MILLIS).build();
-		httpClient = HttpClients.custom().setUserAgent("/Lutung-0.1")
+		httpClient = HttpClients.custom()
+				.setKeepAliveStrategy(new ConnectionKeepAliveStrategy() {
+					@Override
+					public long getKeepAliveDuration(HttpResponse httpResponse, HttpContext httpContext) {
+						return 1;
```

Comment:
why not something longer? I suggest 1000

Reply:
or make it configurable

Reply:
I think the lower is better to not give it a chance to persist. Think of the situation there it spams mandrill many times per second, it persists for another 1000ms and it never ends. To make it configurable we have to make it not a static initialization at least, it is a larger modification. Also adding a parameter to DAS.

Reply:
I'm OK with 1000 and not configurable.

Reply:
fixed